### DISCUSSION TITLE
提醒一下，vercel.app好像被DNS污染了

### DISCUSSION THREAD

**Discussion by StarWEB890** [walinejs/waline#1328] [Q&A] [answered]
今天我访问vercel.app提示如下：
![msedge_1bT3aaBbB3](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_1bT3aaBbB3.20wrrmsxsqps.webp)
我怀疑vercel被墙了，于是我在Q群提醒：
![TIM_UmQItFM7lo](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/TIM_UmQItFM7lo.2k0fzbvmjai0.webp)
然后我在官网看到了这条评论：
![msedge_QtyfJ7Zbco](https://cdn.staticaly.com/gh/StarWEB890/TuChuang@master/images/msedge_QtyfJ7Zbco.g4usr9ypnao.webp)
~~（这也算是提醒一下了，毕竟我这狗比玩意还冤枉了Waline）~~

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/issues/1318 该 issue 之前就已经置顶过了哈~

**Reply by StarWEB890** [reply]
好的，感谢提醒（我是最近才开始无法访问的）

**Comment by gumpBOBO** [comment]
前面我提了问题绑定自己的域名无法访问，后面我改成国际版，才用了几天然后国际版的tmd又挂了，自己域名的又可以了。。离谱
#1211

**Reply by lizheming** [reply]
国内版的需要绑定已备案域名，国际版的似乎只是说需要绑域名，没说需要备案的。

**Reply by elkan1788** [reply]
对的，国际版本+自定义域名（而且这里不能是CNAME转发）不受任何影响，是能正常运行的。

**Reply by Southern-wood** [reply]
> 对的，国际版本+自定义域名（而且这里不能是CNAME转发）不受任何影响，是能正常运行的。

请问不设置 CHAME 转发的话要怎么绑定自定义域名呢，我使用 CHAME 转发确实不能运行……

**Reply by elkan1788** [reply]
> > 对的，国际版本+自定义域名（而且这里不能是CNAME转发）不受任何影响，是能正常运行的。
> 
> 请问不设置 CHAME 转发的话要怎么绑定自定义域名呢，我使用 CHAME 转发确实不能运行……

直接设置里面绑定自己的域名就可以了

**Comment by pydaxing** [comment]
离了个大谱

---
A big departure
